hedera-node/hedera-network-service-impl/src/test/java/com/hedera/node/app/service/network/impl/test/serdes/MonoContextAdapterSerdesTest.java

```diff
@@ -40,38 +37,28 @@ class MonoContextAdapterSerdesTest {
             777L,
             new ExchangeRates(1, 2, 3L, 4, 5, 6L));
 
-    @Mock
-    private ReadableSequentialData input;
-
-    @Mock
-    private WritableSequentialData output;
 
     final MonoContextAdapterCodec subject = new MonoContextAdapterCodec();
 
     @Test
     void doesntSupportUnnecessary() {
         assertThrows(UnsupportedOperationException.class, () -> subject.measureRecord(SOME_CONTEXT));
-        assertThrows(UnsupportedOperationException.class, () -> subject.measure(input));
-        assertThrows(UnsupportedOperationException.class, () -> subject.fastEquals(SOME_CONTEXT, input));
+        assertThrows(UnsupportedOperationException.class, () -> subject.measure(null));
```

Comment:
This changes the effect of the test, which may not have been intentional...

Reply:
I fix it thank for the comment